crawl-ref/source/tileview.cc

```diff
@@ -1392,6 +1393,12 @@ void tile_apply_properties(const coord_def &gc, packed_cell &cell)
             cell.bg |= TILE_FLAG_TRAV_EXCL;
     }
 
+    if(you.rampaging() && adjacent(you.pos(), gc) && Options.show_rampage_indicator
+       && get_rampage_target(gc - you.pos()) != nullptr)
+    {
+        cell.bg |= TILE_FLAG_RAMPAGE;
+    }
+
     if (!map_bounds(gc))
```

Comment:
As written, we'll crash (I suspect!) if we move too close to the map edge with rampage on. Let's try to avoid that and move our new logic below this `map_bounds` check.